README.rst

```diff
@@ -38,6 +38,7 @@ Usage
   -o, --output_file FILE                   CSV file location.                           [required]
   -u, --url URL                            Elasticsearch host URL. Default is http://localhost:9200.
   -i, --index-prefixes INDEX [INDEX ...]   Index name prefix(es). Default is ['logstash-*'].
+  -D, --doc_type doc_type [doc_type ...]   Document type. Default is ['logstash-*'].
```

Comment:
`doc_type – A comma-separated list of document types to search; leave empty to perform the operation on all types`[[link](http://elasticsearch-py.readthedocs.io/en/2.3.0/api.html#elasticsearch.Elasticsearch.search)]
I am not sure that we need default value for this param.
Please change doc_type to upper case.